docs/content/practices/emulator_setup.md

```diff
@@ -109,22 +109,28 @@ define_path_environment_if_needed
 create_and_patch_emulator
 ```
 
-Pay your attention that you also need to wait until your emulator is fully booted.
+Keep in mind that the emulator must fully boot before running any test. Otherwise the tests will fail because there is still no device ready
+on which they can run.
+
+### Summary
+1. create an `ini` configuration file for the emulator
+2. run your emulator via `avd manager` and 
+3. switch the `ini` file generated in 2. with the one we create in 1.
 
 ## How to run an emulator in a Docker?
 
-Running an emulator in a docker a way easier than manually, because it encapsulates all this logic. If you don't have an
-experience with docker, you can check
-[this guide](https://www.youtube.com/watch?v=zJ6WbK9zFpI) to check the basics.
+Running an emulator in a docker is way easier than manually, because it encapsulates all this logic. If you don't have
+experience with docker, check
+[this guide](https://www.youtube.com/watch?v=zJ6WbK9zFpI) to get familiarized with the basics.
 
-There are some popular already built docker images for you:
+There are some popular docker images already built for you:
 
 * [Official Google emulator](https://github.com/google/android-emulator-container-scripts)
 * [Agoda emulator](https://github.com/agoda-com/docker-emulator-android)
 * [Avito emulator](https://hub.docker.com/r/avitotech/android-emulator-29)
 
-Talking about [Avito emulator](https://github.com/google/android-emulator-container-scripts), it also patches your
-emulator with adb commands to prevent tests flakiness and to speed them up
+Talking about the [Avito emulator](https://github.com/google/android-emulator-container-scripts), it also patches your
+emulator with adb commands to prevent tests flakiness and to speed them up.
```

Comment:
would be good to quickly mention how it does it.
From the description here, I take for granted that Avito emulator is the best out of the 3 here? If yes, should we stress the point in here?

What are the differences among them?

As a learner, I want to use the best for my needs